app/components/system_header/index.tsx

```diff
@@ -64,13 +71,23 @@ const SystemHeader = ({isMilitaryTime, createAt, theme, user}: Props) => {
                     testID='post_header.display_name'
                 />
             </View>
-            <FormattedTime
-                timezone={userTimezone!}
-                isMilitaryTime={isMilitaryTime}
-                value={createAt}
-                style={styles.time}
-                testID='post_header.date_time'
-            />
+            <View style={styles.timeContainer}>
+                <FormattedTime
+                    timezone={userTimezone!}
+                    isMilitaryTime={isMilitaryTime}
+                    value={createAt}
+                    style={styles.time}
+                    testID='post_header.date_time'
+                />
+                {isEphemeral && (
```

Comment:
This only solves this for system messages. Wouldn't you need to add something similar to `app/components/post_list/post/header/header.tsx`?

Reply:
Okay, I have added this for `app/components/post_list/post/header/header.tsx`. Let me know the sequence for visible text, should it show after the time or after `HeaderReply` component.

Reply:
Do you mind sharing a screenshot of how it looks right now with your changes? I think it is fine the way it is now, since (I think) ephemeral posts cannot be with priority nor have replies. But with a screenshot it will be more easy for our UX team to verify if it looks as they expect.

Reply:
okay, I have added screenshot of design into the PR description.